Modules/_cursesmodule.c

```
@@ -716,9 +716,14 @@ PyCursesWindow_New(WINDOW *win, const char *encoding)
 static void
 PyCursesWindow_Dealloc(PyCursesWindowObject *wo)
 {
-    if (wo->win != stdscr) delwin(wo->win);
-    if (wo->encoding != NULL)
+    if (wo->win != stdscr) {
+        // Silently ignore errors in delwin(3) (e.g., passing
+        // a NULL pointer results in delwin() returning ERR).
```

I would prefer to check that `wo->win` is not NULL rather than documenting the error.

---

Since there are other possible errors, I'll keep the comment about ignoring errors. The manual says:

> returns an error if the window pointer is null, or if the window is the parent of another window. This implementation also maintains a list of windows, and checks that 
the pointer passed to delwin is one that it created, returning an error if it was not.

---

I'm only referring to "(e.g., passing a NULL pointer results in delwin() returning ERR)" part of your comment.

---

Sorry, I wasn't clear: I agree with you so I only kept the "Silently ignore errors in delwin(3)" part.